JavaScript/tests/syntax_test_typescript.ts

```diff
@@ -833,7 +833,7 @@ if (a < b || c < d) {}
 //             ^ keyword.operator.logical
 
 const f = (): any => {};
-//    ^ entity.name.function meta.binding.name variable.other.readwrite
+//    ^ meta.binding.name entity.name.function variable.other.readwrite
 
```

Comment:
Note that to test this PR, there should be a second test line that checks for `- entity.name.function` after the function name.

Reply:
Good point; added.